src/plugins/data/common/field_formats/converters/string.ts

```diff
@@ -6,14 +6,15 @@
  * Side Public License, v 1.
  */
 
+import escape from 'lodash/escape';
 import { i18n } from '@kbn/i18n';
 import { asPrettyString } from '../utils';
 import { KBN_FIELD_TYPES } from '../../kbn_field_types/types';
 import { FieldFormat } from '../field_format';
-import { TextContextTypeConvert, FIELD_FORMAT_IDS } from '../types';
+import { TextContextTypeConvert, FIELD_FORMAT_IDS, HtmlContextTypeConvert } from '../types';
 import { shortenDottedString } from '../../utils';
 
-export const emptyLabel = i18n.translate('data.fieldFormats.string.emptyLabel', {
+const emptyLabel = i18n.translate('data.fieldFormats.string.emptyLabel', {
```

Comment:
:information_source: Removed the `export`, since we're not using this outside this file.